confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/event/ConfluenceFilteringEvent.java

```diff
@@ -61,4 +79,13 @@ public Collection<Long> getDisabledSpaces()
     {
         return disabledSpaces == null ? Collections.emptyList() : disabledSpaces;
     }
+
+    /**
+     * @return the renamed spaces
+     * @since 9.88.5
+     */
+    public Map<String, String> getRenamedSpaces()
+    {
+        return renamedSpaces == null ? new HashMap<>() : renamedSpaces;
+    }
```

Comment:
```suggestion

    /**
     * @return the space targets
     * @since 9.88.5
     */
    public Map<String, String> getSpaceTargets()
    {
        return spaceTargets;
    }
```